### DISCUSSION TITLE
How to bulk register 1000 user accounts with specified username, realnames & emails?

### DISCUSSION THREAD

**Discussion by mawann** [mastodon/mastodon#20447] [Q&A]
I'm teacher managing a Mastodon instance. This instance is not accepting registrations. No one can register unless registered by me. I want to register all my students into this instance, with the specified usernames, real names & emails. They are not allowed to choose a username, **so they are responsible for their writing**. Can this be done? How to?

**Comment by MrGeneration** [comment]
You're loking for `tooctl accounts create` here:
https://docs.joinmastodon.org/admin/tootctl/#accounts

You'll have to loop through your list via script I guess. There's no API methods that allow this step if I've seen it correctly.

**Reply by mawann** [reply]
Thank you
I'm starting to understand.
Is this how I should type?

```
su - mastodon
RAILS_ENV=production live/bin/tootctl accounts create user1 --email=user1@example.com --confirmed
RAILS_ENV=production live/bin/tootctl accounts create user2 --email=user2@example.com --confirmed

```
The next question is: How do I get users approved right away?

**Reply by MrGeneration** [reply]
Right, the above commands look fitting.

The `--confirmed` flag is already approving the users. There's no second step required - not even by the user itself.
